confed/editor.go

```diff
@@ -228,9 +228,12 @@ func (editor *Editor) Load(args *EditorPathArgs, reply *EditorContentResponse) e
 		wbgong.Error.Printf("Failed to read config file %s: %s", schema.PhysicalConfigPath(), err)
 		return invalidConfigError
 	}
+	if len(bs.preprocessErrors) != 0 {
+		wbgong.Warn.Printf("Load config %s: %s", schema.PhysicalConfigPath(), bs.preprocessErrors)
```

Comment:
Отсюда как будто не очень понятно, что выводится stderr из препроцессора.
```suggestion
		wbgong.Warn.Printf("config preprocessor of %s printed in stderr: %s", schema.PhysicalConfigPath(), bs.preprocessErrors)
```

а лучше ещё и команду показать, если она выше по логам не показывается.

Кстати, а с длинным выводом оно хорошо справляется? как это потом в journald выглядит? Не надо ли разбить построчно на всякий случай?

Reply:
![image](https://github.com/wirenboard/wb-mqtt-confed/assets/86825564/6ed1b2f8-9b87-48c6-beea-42e511882607)

Reply:
Может, мы его тогда с новой строки отдельными записями в лог и выгрузим (можно с каким-нибудь префиксом, чтобы отличать от логов конфеда)? а то это не очень читаемое получилось